fastchat/train/train_lora.py

```diff
@@ -170,12 +175,19 @@ def train():
         model=model, tokenizer=tokenizer, args=training_args, **data_module
     )
 
+    print(f"Training data has {len(data_module['train_dataset'])} examples")
+    print(f"Eval data has {len(data_module['eval_dataset'])} examples")
+
+    if torch.cuda.device_count() > 1:
```

Comment:
I believe the code from lines `152-155` achieves this, so I think we can delete this.